backend/src/main/kotlin/ch/learntrack/backend/backoffice/user/UserResource.kt

```diff
@@ -16,4 +21,18 @@ public class UserResource(private val userService: UserService) {
     @PostMapping("/create")
     public fun createAdminUser(@RequestBody createUserDto: CreateUserDto): Unit =
         userService.createAdminUser(createUserDto)
+
+    @GetMapping("/getAllAdminsForSchool")
+    public fun getAllAdminsForSchool(@RequestParam schoolId: UUID): List<UserDto> = userService.getAllAdminsForSchool(
+        schoolId,
+    )
+
+    //please someone who is smarter than me (me=Shay) fix this
+    @GetMapping("/getTeacherCounts")
```

Comment:
```suggestion
    @GetMapping("/getTeacherCount")
```
at least still don't like it but its okay for the moment.